docs/node/week-4/homework.md

```diff
@@ -0,0 +1,8 @@
+---
+id: homework
+title: Coursework
+sidebar_label: Coursework
+---
+
+All of the coursework for this week can be found in the Node Homework repository [here](https://github.com/CodeYourFuture/NodeJS-Homework) under the Week 4 folder
```

Comment:
I don't currently see a week 4 folder, or a PR adding one - am I missing this, or is happening in the future? Do we want to block this PR on adding that?

Reply:
> I don't currently see a week 4 folder, or a PR adding one - am I missing this, or is happening in the future? Do we want to block this PR on adding that?

This is similar to #620 and #638. Week 4 will need to be linked, could you help with that @illicitonion?

Reply:
For getting the node week 4 page linked, hopefully you can crib from https://github.com/CodeYourFuture/syllabus/pull/638 - you need to edit sidebars.js to add to the left hand side, and docs/node/index.md to add a new table row. (If you're having trouble, I can help out, but hopefully you can crib that ok yourself :))

But the comment I was leaving here is a bit different - the other repo (https://github.com/CodeYourFuture/NodeJS-Homework) doesn't have a "week 4" folder - I'd expect us to add one there before merging this PR (or alternatively, to remove the homework.md in this PR, and only add it in a future PR when the homework exists). What do you think?

Reply:
Alright, I'll try to crib your example to link the pages. Let's see how I fare :)

Re homework repo: Sorry, now I understand what you meant. We have had the same situation for JS3, right? [The JS3 homework repo](https://github.com/CodeYourFuture/JavaScript-Core-3-Homework) doesn't have a week 4, but we have already merged [PD week 4 PR](https://github.com/CodeYourFuture/syllabus/pull/620).

I think this is happening because syllabus pages for JS3, React, Node, and SQL still have only 3 weeks. In practice, all modules take place over 4 weeks. Again in practice, SQL is called databases... 

If adding a folder is as straightforward as copying an existing example, I'm happy to add week 4 folder to those modules (I can try to crib the NodeJS repo).

Reply:
Update: maybe I was looking to the wrong place. There seems to be a repo for [JS3 Week 4](https://github.com/CodeYourFuture/JavaScript-Core-3-Coursework-Week4), as well as for [Node Week 4](https://github.com/CodeYourFuture/Node-Coursework-Week4).

But I'm not sure what this means...